README.md

```diff
@@ -974,7 +974,7 @@ https://twitter.com/SilentImp
 
 Artem Yavorsky  
 Topics: Babel, ECMAScript, Compilers  
-https://twitter.com/yavorsky_
+[https://twitter.com/yavorsky_](https://twitter.com/yavorsky_)
```

Comment:
In other sections, there are not Markdown links. As I know there's no need to wrap such links in `[]()`, since simple URLs are automatically converted to clickable ones.

Reply:
The Markdown parser doesn't understand underscores at the end of links :(